### PR TITLE
Add new package

### LINKED ISSUE #35: Implement add new package pages (same as exiting packages, just empty)

### PR DESCRIPTION
**Pull request by ktkization**
closes #35

### CODE REVIEW

File: webapp/lib/model.dart [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import 'view.dart' as view;
+
```
Comment (marianamarasoiu) [inline webapp/lib/model.dart]: unused import?

File: webapp/lib/controller.dart [hunk 2]
```diff
@@ -140,24 +145,12 @@ void loadDashboardView() {
...
-        'Ask a question with fixed answers',
-        {'Needs' : 'Q/A, Labelling team, Safeguarding response', 'Produces' : 'Dashboard for distribution of answers'}),
-      new view.AvailablePackagesViewPartial('Information Service',
-        'Answer people\'s questions',
-        {'Needs' : 'Response protocol, Labelling team, Safeguarding response', 'Produces' : 'Thematic distribution, work rate tracker'}),
-      new view.AvailablePackagesViewPartial('Bulk Message',
-        'Send set of people a once off message',
-        {'Needs' : 'Definition of who. Safeguarding response', 'Produces' : 'Success/Fail tracker'})
-    ]);
+  for (var data in model.activePackages) {
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: nit: rename `data` to `package`

File: webapp/lib/controller.dart [hunk 3]
```diff
@@ -140,24 +145,12 @@ void loadDashboardView() {
...
+    dashboardView.activePackages.add(new view.ActivePackagesViewPartial(data['name'], data['conversationsLink'], data['configurationLink'], data['chartData']));
+  }
+  for (var data in model.availablePackages) {
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: nit: rename `data` to `package`

**Review by marianamarasoiu — APPROVED** [review]
nice - just a few nits, thanks!

File: webapp/lib/model.dart (hunk 1 above)
Comment (ktkization) [reply webapp/lib/model.dart]: Yes. Removed

File: webapp/lib/controller.dart (hunk 2 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: Changed.

File: webapp/lib/controller.dart (hunk 3 above)
Comment (ktkization) [reply webapp/lib/controller.dart]: Changed.

File: webapp/lib/controller.dart [hunk 4]
```diff
@@ -176,6 +169,18 @@ loadProjectConfigurationView() {
...
 }
 
+void addPackage(String packageName) {
+  model.packageConfigurationData[packageName] = new model.Configuration()
+    ..availableTags = model.tags
+    ..hasAllTags = {}
+    ..containsLastInTurnTags = {}
+    ..hasNoneTags = {}
+    ..suggestedReplies = []
+    ..addsTags = {};
```
Comment (marianamarasoiu) [inline webapp/lib/controller.dart]: After the changes to `model.Configuration`:
```suggestion
    ..availableTags = model.tags;
```

**Review by marianamarasoiu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**